src/python/update_container_definitions.py

```diff
@@ -68,6 +68,10 @@ def run(previous_task_definition, container_image_name_updates, container_env_va
 
     # Expected format: container=...,image-and-tag|image|tag=...,container=...,image-and-tag|image|tag=...,
     try:
+        if "container=" not in container_image_name_updates:
```

Comment:
Could you modify this check such that an error is not raised when the parameter is empty? I ran the build and 2 of our integration tests that pass in an empty `container-image-name-updates` parameter failed (see: https://circleci.com/gh/CircleCI-Public/aws-ecs-orb/2006)